box/schema_test.go

```diff
@@ -0,0 +1,16 @@
+package box
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewSchema(t *testing.T) {
```

Comment:
You could probably add a unit-test that a Schema.User() returns a SchemaUser object with the given doer (with a mock doer). Something like:
```
doer := struct{
    tarantool.Doer
}{}
require.Equal(*NewSchema(doer).User(), *NewSchemUser(doer))
```

Reply:
i added such test for schema 

fixed